### PR TITLE
remove some test utils that are not used

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Playwright.Tests/BaseTests/PlaywrightAssert.cs [hunk 1]
```diff
@@ -48,7 +46,5 @@ internal static class PlaywrightAssert
                 return t;
             }
         }
-
-        internal static void DebugLog(string text) => TestContext.Progress.WriteLine(text);
```
Comment (pavelfeldman) [inline src/Playwright.Tests/BaseTests/PlaywrightAssert.cs:52]: Let's keep this one please, we use it for debugging.

**Comment by pavelfeldman** [comment]
All these utility classes are copied from the main package. We should figure out something to reuse originals rather than further diverge them.

File: src/Playwright.Tests/BaseTests/PlaywrightAssert.cs (hunk 1 above)
Comment (SimonCropp) [reply src/Playwright.Tests/BaseTests/PlaywrightAssert.cs]: done

**Comment by avodovnik** [comment]
Some of this has already been removed in the recent changes, so I'm closing this PR as stale. Sorry about that.

**Pull request closed without merging**